tuna/miopen/fin_eval.py

```diff
@@ -63,14 +63,17 @@ def check_gpu(self):
     """Function to check gpu heartbeat"""
     for _ in range(5):
       if self.machine.chk_gpu_status(self.gpu_id):
-        return
+        return True
       self.logger.warning(
           'Unable to detect GPU: %s, sleeping for %s seconds before retry',
           self.gpu_id, 30)
-      sleep(30)
+      #sleep(30)
```

Comment:
I updated this to return false, meaning quite the tuna process, since we can no longer reset the machine in the barrier. I'll just clean up these comments.